### ISSUE TITLE
Request: make unison modes more consistent+

### ISSUE THREAD

**Issue by M0ahgra** [johnnesky/beepbox#93]
- y not allow unison modes to b used on instrument types (pulse width, spectrum, fm)?
- supersaw could b a unison mode instead of a separate instrument type (potentially controversial)

**Comment by Mid-The-Modder** [comment]
Yeah, the supersaw had a unique purpose and that was to have this customizable chorus-like synthesis. 

For instruments that do not currently support unisons, for everything up to FM/Drumset (drumset might actually also fall in this category, havent looked into it) it is actually fairly easy to implement unisons into them. However, the pulse width's would be quickly overshadowed by the supersaw which can already do said unisons with note placement and setting adjustments, spectrum's is barely audible unless the unison makes sounds that are like chords/harmonization, and noise is a drum channel instrument meaning the need for it there may not be as grand.

Either way, if you want to see what PWM, spectrum, and noise unisons are like, you can check out [UltraBox](https://ultraabox.github.io/)!
